src/main/java/ymfc/storage/Storage.java

```diff
@@ -78,12 +80,18 @@ public void saveIngredients(IngredientList ingredients) throws IOException {
      * @param ingredients Object for user's ingredients.
      * @param ui Object to access the UI class.
      * @param storage Object to access the Storage class.
-     * @throws FileNotFoundException If save file does not exist.
      */
     public void loadRecipes(RecipeList recipes, IngredientList ingredients,
-                            Ui ui, Storage storage) throws FileNotFoundException {
-        File tasksFile = new File(saveRecipeFilePath);
-        Scanner reader = new Scanner(tasksFile);
+                            Ui ui, Storage storage) {
+        Scanner reader;
+        try {
+            File tasksFile = new File(saveRecipeFilePath);
+            reader = new Scanner(tasksFile);
+        } catch (FileNotFoundException exception) {
+            logger.log(Level.INFO, "No recipes savefile found");
+            return;
+        }
+
```

Comment:
Splitting the log messages to the individual recipe and ingredient load methods makes more sense and fixes the bug. Nice work.